rabbitmq-quickstart/rabbitmq-quickstart-producer/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,55 @@
+####
```

Comment:
I believe we can remove the docker files.

Reply:
Docker files are use when running the qs in packaged and native mode because dev services is supported and the apps needs to run on the same network; it reduces the complexity.